disnake/message.py

```diff
@@ -2128,13 +2202,98 @@ async def reply(
             reference = self
         return await self.channel.send(content, reference=reference, **kwargs)
 
-    def to_reference(self, *, fail_if_not_exists: bool = True) -> MessageReference:
+    async def forward(
+        self,
+        content: Optional[str] = None,
```

Comment:
Yes, that's correct. Should we make it possible to pass flags or not?

Reply:
GitHub didn't show this comment while re-reviewing, I inadvertently commented a half-response to this here: https://github.com/DisnakeDev/disnake/pull/1188#discussion_r1845010477.
Ultimately I would prefer we didn't support additional fields here, both for simplicity and because the allowed subset isn't documented atp. They can always be provided to `.send` directly if needed.